app/controller/sdl/RCModulesController.js

```diff
@@ -847,6 +856,21 @@ SDL.RCModulesController = Em.Object.create({
                     dataToReturn.seatControlData = seatControlData;
                 }
             }
+          case 'OBSSH':
+            {
+              if(location_name in this.obsshModels) {
+                  var obsshControlData = this.obsshModels[location_name].setObsshControlData(data.params.moduleData.obsshControlData);
+
+                  if(obsshControlData) {
+                    if (Object.keys(obsshControlData).length > 0) {
+                      FFW.RC.onInteriorVehicleDataNotification({moduleType:'OBSSH', moduleId: moduleUUId,
+                        obsshControlData: obsshControlData});
+                    }
+                    dataToReturn.obsshControlData = obsshControlData;
+                  }
+                }
+              break;
```

Comment:
@YarikMamykin I believe this break is not necessary, but in `case 'SEAT'` one should be added.